### PR TITLE
scala 2.12.1

### PR DESCRIPTION
**Pull request by armanbilge**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/scala.rb [hunk 1]
```diff
@@ -17,17 +17,17 @@ class Scala < Formula
   option "with-docs", "Also install library documentation"
   option "with-src", "Also install sources for IDE support"
 
-  depends_on :java => "1.8+"
+  depends_on java: "1.8+"
```
Comment (woodruffw) [inline Formula/scala.rb]: Please revert this to meet our style checks:

```
== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/scala.rb ==
C: 20: 14: Use hash rockets syntax.
```

Comment (armanbilge) [inline Formula/scala.rb]: Hold on—I got the exact opposite message when I ran `brew audit --strict foo` which is why I made the change in the first place! Any idea why that is?
```
$ brew audit --strict scala
scala:
  * C: 20: col 14: Use the new Ruby 1.9 hash syntax.
Error: 1 problem in 1 formula
```

Comment (armanbilge) [inline Formula/scala.rb]: I've reverted for now though.

**Comment by woodruffw** [comment]
Thanks! Could you squash down to one commit? After that this should be :green_apple:

**Comment by armanbilge** [comment]
Done 👍

**Comment by woodruffw** [comment]
Thanks @armanbilge!

File: Formula/scala.rb (hunk 1 above)
Comment (woodruffw) [inline Formula/scala.rb]: Strange. Possibly a `.rubocop.yml` file not being loaded locally? I'll see if I can reproduce the problem later.

**Pull request closed without merging**